src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CspParameters.Unix.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Security.Cryptography
+{
+    partial class CspParameters
+    {
+        private static void ThrowKeyPasswordNotSupported()
+        {
+            throw new PlatformNotSupportedException("KeyPassword is not supported on this platform.");
+        }
+
+        [CLSCompliantAttribute(false)]
+        public SecureString KeyPassword
+        {
+            get
+            {
+                ThrowKeyPasswordNotSupported();
```

Comment:
CspParameters should already be fully not working on Unix...  At the very least any ctor which can accept one will be throwing.  So this isn't worth splitting.